test/org/apache/catalina/valves/TestSSLValve.java

```diff
@@ -314,12 +352,14 @@ public void testSslCipherUserKeySizeHeaderPresent() throws Exception {
 
     @Test(expected = NumberFormatException.class)
     public void testSslCipherUserKeySizeHeaderBadFormat() throws Exception {
+        setUp();
         mockRequest.setHeader(valve.getSslCipherUserKeySizeHeader(), "not-an-integer");
 
         try {
             valve.invoke(mockRequest, null);
         } catch (NumberFormatException e) {
             Assert.assertNull(mockRequest.getAttribute(Globals.KEY_SIZE_ATTR));
+            mockRequest.setHeader(valve.getSslCipherUserKeySizeHeader(), null);
```

Comment:
Due to the random ordering of the execution of the tests, I had to reset the header at this point because other tests were throwing NumberFormatException.